scalding-core/src/main/scala/com/twitter/scalding/DescribedExecutionApp.scala

```diff
@@ -0,0 +1,57 @@
+package com.twitter.scalding
+
+sealed abstract class ArgOptions
+
+object RequiredArg extends ArgOptions
```

Comment:
case object and sealed abstract trait are normally used here.

Should these all be private?